Tests/SentryTests/SentrySDKTests.swift

```diff
@@ -95,7 +95,7 @@ class SentrySDKTests: XCTestCase {
 
         givenSdkWithHubButNoClient()
 
-        if let autoSessionTracking = SentrySDK.currentHub().installedIntegrations().first(where: { it in
+        if let autoSessionTracking = SentrySDKInternal.currentHub().installedIntegrations().first(where: { it in
```

Comment:
Somewhat redundant, but we should test SentrySDKInternal and SentrySDK (public API) in different tests